modules/primo/includes/primo.search.inc

```diff
@@ -281,14 +281,18 @@ function primo_search_sort_options() {
  *
  * @return array $items
  *   Items similar to the query.
+ *
+ * @throws \Primo\Exception\TransferException
```

Comment:
I think it would be good form to actually catch and log the exception here and just return an empty result.

The caller would and should not know what to do about a `Primo` exception.

Reply:
That's what's happening. This exception is from the Primo client which already handles logging. :)